src/App.css

```diff
@@ -19,6 +20,12 @@
   color: white;
 }
 
+.App-header h1,
+.App-header h3 {
```

Comment:
You might ask yourself:
- Do I want ALL h1 and h3 headers to appear this way, throughout the page? If so, you might remove the ".App-header" prefixes
- Do I want ALL of those headers only within the "App-header" element to appear this way? If so, keep it as it is
- Do I only want these two, specific elements to appear this way? Then you might replace what is here, with a class name, and apply it to your elements. For example...

Here:
```
.Header-custom-spacing {
```
(or whatever you decide to name it)

Then in App.js:
```
        <h1 className="Header-custom-spacing">
          Eric F. Olsen
        </h1>
        <h3 className="Header-custom-spacing">
          Software Engineer
        </h3>
```

Reply:
The choice is up to you. You can certainly start one way, and change later. You'll just need to remember this, as you add more elements, and see if you keep your decision, or change.

Reply:
I'll leave it as-is for now. It'll keep this formatting exclusive to the app-header which I only expect to affect this first Name and Title. If I find next time I create a header that I prefer the spacing I'll move this to be global.

Reply:
@Azmaeus I agree with /All?[ea]n/ but will add, that if you decided to remove the `.App-` from the `h1`/`h3` then you should also move it up to the index.css file instead of this file.

(Sorry I couldn't remember how you spell your name @allProgramming :laughing: )